financial-connections/src/main/java/com/stripe/android/financialconnections/di/FinancialConnectionsSheetNativeModule.kt

```diff
@@ -24,16 +21,4 @@ internal class FinancialConnectionsSheetNativeModule {
     fun providesNavigationManager() = NavigationManager(
         CoroutineScope(SupervisorJob() + Dispatchers.Default)
     )
-
```

Comment:
Is this `UpdateManifest` supposed to be replaced to the new `UpdateManifest` defined in `FlowCoordinatorMessage`?

Reply:
Yeah, for simplicity I removed the abstraction and screens can directly talk to `FlowCoordinator` to send messages.

This is an example of the flow we're using to communicate between the activity (that controls global flow state) and per-screen composables (responsible just for screen specific logic):

1. Hit next in Consent screen -> consent agreed API call triggers, and returns a `Manifest` instance
2. Consent screen sends a `FlowCoordinatorMessage` to update the global manifest with the received, updated instance
3. Parent activity receives message -> updates global state with fresh manifest
4. Consent screen sends a `FlowCoordinatorMessage` to request navigation to next step
3. Parent activity receives message -> decides where to navigate next based on global state manifest (that was updated in 3)

Reply:
To simplify even more, removed all FlowCoordinator communication abstractions so that screens always talk directly to the coordinator - https://github.com/stripe/stripe-android/pull/5140/commits/41a65498a27c4669e769c52cf0ed521fe25ec51f